content/git/concepts/branch/branch.md

```diff
@@ -36,15 +36,44 @@ git branch -d <branch-name>
 
 As long as the branch has no uncommitted code, this command will remove the branch entirely. It will no longer be available to check out, and the changed code will be unrecoverable.
 
+Sometimes, Git will refuse to delete a branch because it has unmerged changes in an existing upstream or remote version of that branch. In such cases, it can be forcefully deleted with either the `-d --force` or capitalized `-D` flag, like so in the `my-branch` examples below.
+
+Option 1:
+
+```shell
+git branch -d --force my-branch
+```
+
+Option 2:
+
+```shell
+git branch -D my-branch
+```
+
+Performing a force delete tells Git the user understands that all changes on that branch will be lost forever, and they want to delete the branch anyway.
+
```

Comment:
Could we also add a short bit about removing the remote branch equivalent (if one exists)?

```
git push <name-of-remote> --delete <name-branch>
```

Not sure if it should go here b/c it deals with branching or somewhere in `push.md` since this uses the `push` command.

Reply:
Holding off on this one for now. In the future, it might be worth adding, perhaps side by side with the way to do it on GitHub.